### PR TITLE
Adding contribution file

### PR DESCRIPTION
**Pull request by OfirBachar**
Adding contribution.md file

### CODE REVIEW

**Review by patrisiakaplun — APPROVED** [review]
LGTM

**Review by idoyek — APPROVED** [review]
LGTM

File: contribution.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+1. Allways provide constructive feedback
+2. Be precise about what needs to be improved
+3. Visualize the bigger picture
+
+# Code Testing
+
+This project uses Pytest as its test framework.
+
+* [Pytest - Quick guide](https://docs.pytest.org/en/7.2.x/getting-started.html)
+* Please note that Pytest is already installed in this project's virtual environment.
```
Comment (Yarboa) [inline contribution.md:62]: No pytest here, at this status

File: contribution.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+**Note: If your PR does not meet one of the requirements, the PR will not be reviewed!**
+
+## Issue a pull request:
+1.	Set yourself as an Assignee
+2.	Try to make your changes small as you can
+3.	Write useful descriptions and titles
+4.	Ask cotributors to review your PR.
+5.	Once you have the sign-off of two developers and one of the maintainers, your PR will be merged.
+
+## Pull request template:
```
Comment (Yarboa) [inline contribution.md:44]: Template is not important

You should describe what should be in the description

File: contribution.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+4.	Issue a pull request and ask us to review it. Don't forget to link the PR to a relevant issue if it exists, **If not CREATE it!**
+5.	Make sure your code is conformed to the PEP8 standard
+
+**Note: If your PR does not meet one of the requirements, the PR will not be reviewed!**
+
+## Issue a pull request:
+1.	Set yourself as an Assignee
+2.	Try to make your changes small as you can
+3.	Write useful descriptions and titles
+4.	Ask cotributors to review your PR.
```
[the remark this answers is not in the document]
Comment (Yarboa) [reply contribution.md:41]: Consider this, (please make it more friendly)

Create PR
1) Clear subject
2) Clear description, 
3) AddResolve #Issue
4) Assign PR to yourself

File: contribution.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+5.	Once you have the sign-off of two developers and one of the maintainers, your PR will be merged.
```
Comment (Yarboa) [inline contribution.md:42]: Consider the foolowing
move to -> \#\# Review Process

And describe the minimum for fast attention 
1) Verify Githab actions tests status = PASSED
2) Select Reviewers under Reviewers section

Note: at least 2 LGTM, Approves needed for Core reviewers to review

File: contribution.md [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+
+## Pull request template:
+1. Making changes/Creating XFILE
+2. Empty line
+3. What, why & how?
+4. Testing
+5. Add Screenshots(if needed)
+
+
+## Reviewing PRs:
```
Comment (Yarboa) [inline contribution.md:52]: Please also explain the difference between Reviewers and CoreReviers

**Review by Yarboa — CHANGES_REQUESTED** [review]

File: contribution.md
Comment (OfirBachar) [inline contribution.md:1]: Deleting unnecessary lines/spaces

**Review by patrisiakaplun — APPROVED** [review]
LGTM

**Review by TalReinfeld — APPROVED** [review]
LGTM

**Review by OfirBachar — COMMENTED** [review]
LGTM

**Review by Yarboa — COMMENTED** [review]
Once you changed 
1. Reviewers
2.  The Maintainers 

Please use it in the whole doc.
Please replace mentors with something like this 

https://github.com/redhat-beyond/Pro4U/pull/16#discussion_r1145285200

**Review by idosinger98 — APPROVED** [review]
LGTM

File: contribution.md [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+1.  Clear subject
+2.  Clear description
+3.  AddResolve #Issue
+4.  Assign PR to yourself
+5.  Ask cotributors and reviewers to review your PR.
+
+## Reviewing PRs
+1.  Verify Githab actions tests status = PASSED
+2.  Select Reviewers under Reviewers section
+**Note: at least 2 LGTM, Approves needed for Core reviewers to review**
```
Comment (Yarboa) [inline contribution.md]: I still see it, Please change Core reviewers to maintainers

File: contribution.md [hunk 7]
```diff
@@ -0,0 +1,115 @@
...
+3.  Once you have the sign-off of two developers and one of the maintainers, your PR will be merged.
+
+##  Reviewers and Maintainers
+1.  The reviewers are feedback given by team members.
+2.  The Maintainers are feedback given by the mentors.
```
[the remark this answers is not in the document]
Comment (Yarboa) [reply contribution.md:57]: consider change to this 
are project maintainers
They are final decision makers on which portions of source code go into a build or release

I think this should move to before "Create PR"

**Review by idosinger98 — APPROVED** [review]
LGTM

**Review by OfirBachar — COMMENTED** [review]
LGTM

**Review by TalReinfeld — APPROVED** [review]
LGTM

File: contribution.md [hunk 8]
```diff
@@ -0,0 +1,113 @@
...
+**_NOTE:_** By running this command all changes on the virtual machine will be deleted.
+
+# Pull Request Process
+1.	Fork the repository and create a separate branch regarding your contribution.
+2.	Update the CHANGELOG file with details of changes to the interface, this includes new environment variables, exposed ports, useful file locations, and container parameters.
+3.	Changes to README file should be commited **ONLY** when there are changes relevant to the essence of the project, or the ways to deploy it!
+4.	Issue a pull request and ask us to review it. Don't forget to link the PR to a relevant issue if it exists, **If not CREATE it!**
+5.	Make sure your code is conformed to the PEP8 standard
+
+**Note: for fast maintainers review please refer Reviewing PRs**
```
Comment (brochwerger) [inline contribution.md:33]: Keep terminology consistent - stick to "Reviewers" and "Maintainers", no "fast reviewers")

File: contribution.md [hunk 9]
```diff
@@ -0,0 +1,113 @@
...
+1.  Clear subject
+2.  Clear description
+3.  AddResolve #Issue
+4.  Assign PR to yourself
+5.  Ask Reviewers and Maintainers to review your PR.
+
+## Reviewing PRs
+1.  Verify Githab actions tests status = PASSED
+2.  Select Reviewers under Reviewers section
+**Note: at least 2 LGTM, Approves needed for Maintainers to review**
```
Comment (brochwerger) [inline contribution.md:54]: not clear - did you mean two LGTM and two approvals from Reviewers before Maintaners review ?

**Review by TalReinfeld — APPROVED** [review]
LGTM

**Review by idosinger98 — APPROVED** [review]
LGTM

File: contribution.md [hunk 10]
```diff
@@ -0,0 +1,113 @@
...
+
+### Run the project on your local machine
+
+A step by step series that tell you how to get Pro4U up and running
+
+1. Clone the repository
+2. Open command prompt in Pro4U directory
+3. Spin the environment using `vagrant up` 
+4. That's it! your virtual machine is up and running 
+5. Go to http://localhost:8000 in your web browser
```
Comment (guybeckenstein) [inline contribution.md:21]: Note that it is also possible to enter the website using http://127.0.0.1:8000/, but I don't find this note mandatory

**Review by guybeckenstein — APPROVED** [review]

**Review by Yarboa — APPROVED** [review]

**Pull request merged**